packages/data-explorer/src/store/actions.ts

```diff
@@ -26,15 +26,14 @@ export default {
       await dispatch('header/fetchBreadcrumbs')
     }
   },
-  async fetchViewData ({ dispatch, state }: any) {
+  async fetchViewData ({ commit, dispatch, state }: any) {
     if (state.dataDisplayLayout === 'CardView') {
       dispatch('fetchCardViewData')
     } else {
       dispatch('fetchTableViewData')
     }
   },
   fetchCardViewData: async ({ commit, state, getters }: { commit: any, state: ApplicationState, getters: any }) => {
-    commit('setLoading', true)
```

Comment:
Why remove the spinner?

Reply:
Its wrapped around the whole data fetch function, including the fetchTablePermissions & fetchTableMeta